### DISCUSSION TITLE
Do not hide completed habits that haven't been done on that day

### DISCUSSION THREAD

**Discussion by sfsworms** [iSoron/uhabits#984] [Feature Requests]
So, the "hide completed" option is great to keep the screen clear as habits are piling up. 

However, habits that have more complex schedule than "every day" will be hidden even if not performed that day. So if I want to note down that e.g. I ran for the fourth time this week when my habit is set to run three times a week, I need to unhide all habits, find the running habits, check it and hide all again. It also means I'm not reminded that running is something I aspire to do more off.

I'm aware that feature creep goes against the nice, slick, simple esthetic of the app, but I think that an option either to change the behavior of "hide completed" or maybe at the habit level could be integrated without burdening the appearance of the app or its use.

**Comment by hiqua** [comment]
> I think that an option either to change the behavior of "hide completed"

What behavior do you suggest?

FYI this shortcoming of "hide completed habits" is the reason why the sort by status was implemented: with this, habits to be done appear first, then habits that are not due but also not done, then done habits. For your example, running would end up in the middle of the list.
